packages/v3/src/esp-entity-table.ts

```diff
@@ -105,9 +105,13 @@ export class EntityTable extends LitElement implements RestAction {
           return a.entity_category < b.entity_category  
             ? -1  
             : a.entity_category == b.entity_category  
-            ? sortA < sortB  
-              ? -1  
-              : 1  
+            ? sortA === sortB  
+              ? a.name.toLowerCase() < b.name.toLowerCase()
+                ? -1
+                : 1  
+              : sortA < sortB
+                ? -1
+                : 1   
```

Comment:
Untested simplification guess.
```suggestion
              ? a.name.toLowerCase() < b.name.toLowerCase() || sortA < sortB
                ? -1
                : 1
```

Reply:
tried it, but like that it allways sorts by the name and ignores the `sorting_weight`.

We need to check `a.name.toLowerCase() < b.name.toLowerCase()` only if `sortA === sortB` so i dont see any furture simplifications

Reply:
I tried it like this 
```ts
        this.entities.sort((a, b) => {
          const sortA = a.sorting_weight ?? a.name;
          const sortB = b.sorting_weight ?? b.name;
          return a.entity_category < b.entity_category
            ? -1
            : a.entity_category == b.entity_category
              ? a.name.toLowerCase() < b.name.toLowerCase() || sortA < sortB
                ? -1
                : 1
            : 1
        });
```
or did you mean it differently?

Reply:
Ah I meant the following, though quite likely it still wouldn't have worked like expected. Thanks for giving it a whirl.
```
            ? sortA === sortB  
              ? a.name.toLowerCase() < b.name.toLowerCase() || sortA < sortB
                ? -1
                : 1  
```